CHANGELOG.md

```diff
@@ -4,7 +4,23 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
-## [v6.4.0](https://github.com/voxpupuli/puppet-systemd/tree/v6.4.0) (2024-02-24)
+## [v6.5.0](https://github.com/voxpupuli/puppet-systemd/tree/v6.5.0) (2024-03-05)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-systemd/compare/v6.4.0...v6.5.0)
+
+**Implemented enhancements:**
+
+- Add possibiliry to setup limits for user sessions [\#417](https://github.com/voxpupuli/puppet-systemd/issues/417)
```

Comment:
I fixed a typo here, but in general I add `skip-changelog` when there is a PR that implements it. I'll leave it up to you whether you fix the typo, or skip it.

Reply:
updated, thanks